lua/which-key/state.lua

```diff
@@ -235,7 +235,14 @@ function M.execute(state, key, node)
   end
   Util.debug("feedkeys", tostring(state.mode), keystr)
   local feed = vim.api.nvim_replace_termcodes(keystr, true, true, true)
-  vim.api.nvim_feedkeys(feed, "mit", false)
+
+  -- NOTE: We want to check the mapping without the repeat quantifier ie. `<leader>j` and not `4<leader>j`
```

Comment:
i am not sure if i missed out anything here though... 

another idea is that we only perform the `has_mapping` check if we're in hydra mode... 
but i might need some help here, not too sure how we can do this